include/ola/Clock.h

```diff
@@ -40,8 +40,10 @@
 
 namespace ola {
 
+static const uint64_t NSEC_IN_SEC = 1000000000;
```

Comment:
Can we reverse the order they're defined and define them based on each other, i.e.:
MSEC_IN_SECONDS = 1000
USEC_IN_SECONDS = MSEC_IN_SECONDS * 1000
etc

Reply:
Better yet, please define them based on ONE_THOUSAND.

Reply:
done